lib/class/Controller.js

```diff
@@ -134,7 +134,41 @@ Controller.prototype.createPage = function createPage(req, res, next) {
 };
 
 Controller.prototype.editPage = function editPage(req, res, next) {
-  console.log('TODO');  next();
+  if (!res.locals.template) {
+    res.locals.template = res.local.model + '/' + 'editPage';
+  }
+
+  console.log(req.method);
+
+  if (req.method === 'GET') {
+    res.responseMethod = 'ok';
+    next();
+  } else if (req.method === 'POST') { //should be PUT or patch
+      var id = req.params.id || (res.locals.record
+                            && res.locals.record.dataValues
+                            && res.locals.record.dataValues.id);
+
+    if (!id) {
+      return res.notFound();
+    }
+
+    console.log(req.body);
+    res.locals.Model.findById(id)
```

Comment:
You dont need this find in controller, by default we.js context loader load record and set it in res.locals.record after run controller fn.
only check if it exists:

```js
if(!res.locals.record) return res.notFound();
```
see: https://github.com/wejs/we-core/blob/master/lib/router.js#L387